packages/router/src/lib/helpers/auction.ts

```diff
@@ -8,3 +8,5 @@ export const AUCTION_EXPIRY_BUFFER = 5 * 60;
  * @returns Expiry time of a given bid in s
  */
 export const getBidExpiry = (currentTime: number) => currentTime + 60 * 5;
+
+export const FEE_CHAIN_IDS = [1, 3, 4, 5];
```

Comment:
If we do this we need to add to deploy script, right?

Reply:
right